app/Models/Signup.php

```diff
@@ -22,7 +22,7 @@ class Signup extends Model
      * @var array
      */
     public static $indexes = [
-        'campaign_id', 'campaign_run_id',
+        'campaign_id', 'campaign_run_id', 'updated_at',
```

Comment:
In order to make this an actual index on the DB it has to be setup in a migration file ([here's an example](https://github.com/DoSomething/rogue/blob/master/database/migrations/2017_03_09_192255_add_signup_id_index_to_events_table.php)) 😄 :pancakes: